mizar/obj/endpoint.py

```diff
@@ -177,6 +178,9 @@ def set_vpc(self, vpc):
     def set_net(self, net):
         self.net = net
 
+    def set_k8s_pod_name(self, k8s_pod_name):
```

Comment:
Is this a temporal design? It should be the other way around:  pod nic status or annotation tracks the id of endpoint.  With current design we won't be able to support multi-nic pods, and also Mizar got coupled with specific computing clients.